cypress/integration/search.spec.js

```diff
@@ -30,6 +30,40 @@ context('Searching', () => {
         .and('contain', 'wâpamêw') // lemma
         .and('contain', 'Action word') // verb
     })
+
+    it('should show linguistic breakdowns as an ordered list when the user clicks on the ? icon beside a word', () => {
+      // begin from the homepage
+      cy.visit('/');
+
+      // lock onto the searchbar
+      cy.get('[data-cy=search]')
+      // get a word (nipaw)
+      .type('nipaw');
+      
+      // tab through the elements to force the tooltip to pop up
+      cy.get('[data-cy=information-mark]').first().click({force:true});
+
+      // see the linguistic breakdown as an ordered list
+      cy.get('[data-cy=linguistic-breakdown]').children().eq(1).should('have', 'li');
```

Comment:
This is fine, but I'm puzzled why this code gets 1-th element (second element in human speech). I think I'd write this using the two argument form of the `contains()` assertion:

```suggestion
      cy.get('[data-cy=linguistic-breakdown]').contains('li', 'Action word');
```